README.md

```diff
@@ -380,6 +378,19 @@ add-zsh-hook chpwd load-nvmrc
 load-nvmrc
 ```
 
+#### bash
+
+Put this into your `$HOME/.bashrc` to call `nvm use` automatically whenever you enter a directory that contains an
+`.nvmrc` file with a string telling nvm which node to use:
+
+```bash
+# place this after nvm initialization!
+cd() {
+	builtin cd "$@"
+	[ $? -eq 0 ] && [ -f ".nvmrc" ] && [ -s ".nvmrc" ] && nvm use
```

Comment:
- if `[ $? -eq 0 ]` fails, what exit code is returned?
 - if `[ -f ".nvmrc" ]`, `[ -s ".nvmrc" ]`, or `nvm use` fails, i would expect them to do nothing more than perhaps log to stderr, and the original `cd` exit code returned.